src/locales/en.json

```
@@ -74,6 +74,7 @@
   "alerts": {
     "clearReset": "This will clear the whole canvas. Are you sure?",
     "couldNotCreateShareableLink": "Couldn't create shareable link.",
+    "couldNotLoadInvalidJSON": "Couldn't load invalid json",
```

```suggestion
    "couldNotLoadInvalidFile": "Couldn't load invalid file",
```

User's don't know about JSON :)